### PR TITLE
move gh-pages into master, setup sbt-site, sbt-ghsite, tut

### PR DESCRIPTION
**Pull request by stew**

### CODE REVIEW

**Comment by timperrett** [comment]
@stew why do some of the .md files live in `src/main/tut` and others live in site? What's the difference?

**Comment by stew** [comment]
the ones in src/main/tut are the ones which are compiled. I suppose they could all live there.

**Comment by timperrett** [comment]
Would make sense to keep it consistent IMHO

**Comment by stew** [comment]
sure, moved them :)

File: docs/build.sbt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+site.addMappingsToSiteDir(tut, "")
+
+ghpages.settings
+
+ghpagesNoJekyll := false
+
+includeFilter in makeSite := "*.yml" | "*.md" | "*.html" | "*.css" | "*.png" | "*.jpg" | "*.gif" | "*.js" | "*.swf"
+
+git.remoteRepo := "git@github.com:stew/remotely.git"
```
Comment (timperrett) [inline docs/build.sbt]: This needs fixing.

**Comment by timperrett** [comment]
Should the HTML output be checked in? That seems wrong doesn't it?

**Pull request merged**